### DISCUSSION TITLE
六分街日报

### DISCUSSION THREAD

**Discussion by yutong903** [share121/inter-knot#2655] [其他]
近日，六分街的两位录像店店主被成功抓获，据朱鸢警官调查，该兄妹二人看似做正经生意，实则偷偷倒卖皇片，朱鸢警官和青衣警官蹲守4个小时，终于将麦片兄妹抓获。
据报道，朱鸢警官先是与录像店店主之一的铃某交好，实则趁机调查二人，终于，经过两个星期的调查，朱鸢警官锁定了他们的接头地点，并顺藤摸瓜，抓出了参与皇片买卖的其他人员。
经调查，此兄妹二人就是大名鼎鼎的传奇绳匠“法厄同”朱鸢警官抓住他们，又立下了大功一件。
那么今天的报道就到这里，我们明天再见。<img src="https://img-static.mihoyo.com/communityweb/upload/8b198c0a467614f263e08593bd4b000d.png" alt="安比_无语" width="75" >

**Comment by 5cheng7** [comment]
审讯妹妹的委托我接了。

**Comment by ysd1911** [comment]
啊？是六分街的Random Play吗？我上次去那里借带子的时候店内两位店长人可好了，还建议我办个会员，他们就这样被抓了？
而且还是传奇绳匠“法厄同”？这也太离谱了吧<img src="https://bbs-static.miyoushe.com/static/2024/03/21/8fe47df50ba7e17e012e8ba5baaaad49_4838310462379389617.png" alt="丽都漫步-安东" width="75" >

**Comment by I-m-Miss-Lucy-s-dog** [comment]
怪不得繩網都沒有法厄童接單的消息 原來是轉去賣皇片了

**Comment by aizai1173** [comment]
那我预购的片谁给我补呢...
